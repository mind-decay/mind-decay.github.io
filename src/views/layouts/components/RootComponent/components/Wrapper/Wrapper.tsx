import { ReactNode } from 'react';

import { GlobalStyles } from 'styles/global';

type WrapperProps = {
  children: ReactNode;
};

export const Wrapper = ({ children }: WrapperProps) => {
  return (
    <>
      <GlobalStyles />
      {children}
    </>
  );
};
