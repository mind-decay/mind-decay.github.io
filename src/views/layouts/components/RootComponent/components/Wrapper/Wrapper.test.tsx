import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import { Wrapper } from './Wrapper';

test('renders wrapper for root component', () => {
  const { container } = render(<Wrapper>test 2</Wrapper>);

  expect(container).toMatchSnapshot();
});
