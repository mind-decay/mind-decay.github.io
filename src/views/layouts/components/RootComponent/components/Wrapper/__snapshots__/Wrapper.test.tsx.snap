// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders wrapper for root component 1`] = `
<div>
  test 2
</div>
`;
